### ISSUE TITLE
Projections highlighted as bound variables in record type definition.

### ISSUE THREAD

**Issue by andreasabel** [agda/agda#1790]
``` agda
record Σ (A : Set) (B : A → Set) : Set where
  constructor _,_
  field
    proj₁ : A
    proj₂ : B proj₁
```

The last `proj₁` should be highlighted as projection, but is highlighted as bound variable.

**Comment by andreasabel** [comment]
There actually is correct highlighting info, but it is overwritten by incorrect highlighting info coming from the record constructor type.  Deleteing the latter for highlighting solves this issue.

**Comment by andreasabel** [comment]
Closed by ccfc2df008c697b73e02f9e83e379a748020efc2.

**Comment by andreasabel** [comment]
Problem persisted for records in a mutual block.
